spec/schemas/_common.query_dsl.yaml

```diff
@@ -623,6 +623,8 @@ components:
         - $ref: '#/components/schemas/QueryBase'
         - type: object
           properties:
+            field:
```

Comment:
Is this right? If I understand correctly this hard-codes the name of the field to `field`.  In your tests, rename `field` to `location`, it will break.

Reply:
This was a fundamental question I had on a previous PR https://github.com/opensearch-project/opensearch-api-specification/pull/427#issuecomment-2231423577. It's not a hard-coded `field`, it's _any_ `geo_point` or `geo_shape` field.

Reply:
So the name can be anything - use `additionalProperties`.